buildkite/scripts/build-hardfork-package.sh

```diff
@@ -58,18 +58,18 @@ _build/default/src/app/runtime_genesis_ledger/runtime_genesis_ledger.exe --confi
 echo "--- Create hardfork config"
 FORK_CONFIG_JSON=config.json LEDGER_HASHES_JSON=hardfork_ledger_hashes.json scripts/hardfork/create_runtime_config.sh > new_config.json
 
-existing_files=$(aws s3 ls s3://snark-keys.o1test.net/ | awk '{print $4}')
```

Comment:
Could you extract change to `ro` bucket to a separate PR?

Reply:
yes i will do that